samples/observability-traces.js

```diff
@@ -0,0 +1,139 @@
+//   title: Observability (Tracing) with OpenTelemetry
+//   usage: node observability-traces.js <PROJECT-ID> <INSTANCE-ID> <DATABASE-ID>
+
+'use strict';
+
+// Setup OpenTelemetry and the trace exporter.
+// [START spanner_trace_and_export_spans]
+const {Resource} = require('@opentelemetry/resources');
+const {
+  NodeTracerProvider,
+  TraceIdRatioBasedSampler,
+  // eslint-disable-next-line n/no-extraneous-require
+} = require('@opentelemetry/sdk-trace-node');
+// eslint-disable-next-line n/no-extraneous-require
+const {BatchSpanProcessor} = require('@opentelemetry/sdk-trace-base');
+const {
+  SEMRESATTRS_SERVICE_NAME,
+  SEMRESATTRS_SERVICE_VERSION,
+  // eslint-disable-next-line n/no-extraneous-require
+} = require('@opentelemetry/semantic-conventions');
+
+const resource = Resource.default().merge(
+  new Resource({
+    [SEMRESATTRS_SERVICE_NAME]: 'spanner-sample',
+    [SEMRESATTRS_SERVICE_VERSION]: 'v1.0.0', // The version of your app running.,
+  })
+);
+
+/*
+ * Uncomment these lines to debug OpenTelemetry.
+const {diag, DiagConsoleLogger, DiagLogLevel} = require('@opentelemetry/api');
+diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ALL);
+*/
+
+// Create the Google Cloud Trace exporter for OpenTelemetry.
+const {
+  TraceExporter,
+} = require('@google-cloud/opentelemetry-cloud-trace-exporter');
+const exporter = new TraceExporter();
+
+// Optionally, you can enable gRPC instrumentation, by removing this guard.
+if (process.env.SPANNER_ENABLE_GRPC_INSTRUMENTATION === 'true') {
+  // eslint-disable-next-line n/no-extraneous-require
+  const {registerInstrumentations} = require('@opentelemetry/instrumentation');
+  const {GrpcInstrumentation} = require('@opentelemetry/instrumentation-grpc');
+  registerInstrumentations({
+    instrumentations: [new GrpcInstrumentation()],
+  });
+}
+
+async function main(
+  projectId = 'my-project-id',
+  instanceId = 'my-instance-id',
+  databaseId = 'my-project-id'
+) {
+  // Create the OpenTelemetry tracerProvider that the exporter shall be attached to.
+  const provider = new NodeTracerProvider({
+    // Trace every single request to ensure that we generate
+    // enough traffic for proper examination of traces.
+    sampler: new TraceIdRatioBasedSampler(1.0),
+    resource: resource,
+  });
+  provider.addSpanProcessor(new BatchSpanProcessor(exporter));
+
+  // Uncomment this to make it a global tracerProvider instead
+  // of passing it into SpannerOptions.observabilityOptions.
+  // provider.register();
+
+  // Create the Cloud Spanner Client.
+  const {Spanner} = require('@google-cloud/spanner');
+
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance-id';
+  // const databaseId = 'my-database-id';
+
+  const spanner = new Spanner({
+    projectId: projectId,
+    observabilityOptions: {
+      // Optional, but you could rather register the global tracerProvider.
```

Comment:
Remove this. As we already added a comment for global trace provider above

Reply:
Done, thanks!